prawcore/auth.py

```diff
@@ -331,23 +331,27 @@ class ScriptAuthorizer(Authorizer):
 
     AUTHENTICATOR_CLASS = TrustedAuthenticator
 
-    def __init__(self, authenticator, username, password):
+    def __init__(self, authenticator, username, password, otp_function=None):
```

Comment:
While the API uses `otp` as the parameter name in the API, I'd prefer if we stick with human readable names. Also I prefer to name such parameters `callback` instead of `function`. As a result, can we instead use `two_factor_callback` here?